.changes/v0.10.0.md

```diff
@@ -0,0 +1,15 @@
+## v0.10.0 - 2024-02-27
+
+
+### Added
+- New [Dagger functions and modules](https://docs.dagger.io/developer-guide/overview/482011/architecture)
```

Comment:
To me, this seems like a better link: https://deploy-preview-6750--devel-docs-dagger-io.netlify.app/guides/457482/create-app-ci-module

This is what the PR is currently linking to: https://deploy-preview-6750--devel-docs-dagger-io.netlify.app/developer-guide/overview/482011/architecture

WDYT @vikram-dagger?

Reply:
Don't think the guide is quite the right one - to quote:

> This guide assumes that:
> - You know the basics of calling and writing Dagger Modules

Maybe we should link to the Quickstart instead? https://deploy-preview-6750--devel-docs-dagger-io.netlify.app/quickstart

Reply:
Updated to point to the quickstart instead - we can later update this if we need to, and modify the github release description.

Reply:
OK!